qa/workunits/fs/misc/multiple_rsync.sh

```diff
@@ -1,16 +1,26 @@
 #!/bin/sh -ex
 
-sudo rsync -av --exclude local/ /usr/ usr.1
-sudo rsync -av --exclude local/ /usr/ usr.2
+
+# Populate with some arbitrary files from the local system.  Take
+# a copy to protect against false fails from system updates during test.
+export PAYLOAD=/tmp/multiple_rsync_payload.$$
+cp -r /usr/lib/ $PAYLOAD
```

Comment:
It looks like we don't actually invoke this script multiple times on the same host in our current test suite, but it would be good if it could take that. Perhaps by creating/checking some lock files?